.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,40 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/ubuntu
 {
-	"name": "Ubuntu",
-	"postCreateCommand": "bash /workspaces/backend.ai/.devcontainer/bootstrap.sh",
-	"forwardPorts": [6022, 8090, 8091]
-}
\ No newline at end of file
+  "name": "Ubuntu",
+  "onCreateCommand": "cd /workspaces/backend.ai && bash /workspaces/backend.ai/scripts/install-dev.sh --codespaces-on-create >> install-oncreate.log 2>&1",
+  "postCreateCommand": "cd /workspaces/backend.ai && bash /workspaces/backend.ai/scripts/install-dev.sh --codespaces-post-create >> install-postcreate.log 2>&1",
```

Comment:
I think it is a bad practice to hide the log. Do we have another way to see the output?

Reply:
Probably we need to skip the completion guide message depending on whether we're on codespace or not, but I think we don't have to omit it.

Reply:
Initial design of this command was to retain installation log to a file (`/workspaces/backend.ai/install.log`). Will it be better to just print installation outputs on stdout? In that way, users can see current status through the option under Command Palette. I'll follow your decision.

Reply:
I think it's better to utilize existing & officially provided interfaces to inspect the logs whenever possible.

Reply:
Gotcha. I'll update the configuration.